contrib/internal/httputil/trace.go

```diff
@@ -53,8 +54,11 @@ func TraceAndServe(h http.Handler, cfg *TraceConfig) {
 	span, ctx := tracer.StartSpanFromContext(cfg.Request.Context(), "http.request", opts...)
 	defer span.Finish(cfg.FinishOpts...)
 
-	cfg.ResponseWriter = wrapResponseWriter(cfg.ResponseWriter, span)
-	httpinstr.WrapHandler(h, span).ServeHTTP(cfg.ResponseWriter, cfg.Request.WithContext(ctx))
+	// Wrap the handler with the appsec instrumentation when enabled
```

Comment:
I think you can remove this comment as it is absolutely obvious what is going on.

Reply:
Fixed